c3po/api/create_app.py

```diff
@@ -0,0 +1,19 @@
+from logging import getLogger
+
+from flask import Flask
+
+from app_config import config_by_name
+
+LOG = getLogger(__name__)
+
+LOG.info('configured logger!')
+
+def create_app(config_name):
+    app = Flask(__name__, template_folder="../static/templates")
```

Comment:
Why is the `template_folder` required?

Reply:
Just in case we ever need to create a form for testing that requires CSRF Validation and we use WTForms. I don't really think it'll be needed though, I'll remove it.